RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -79,17 +79,17 @@ extension DeviceFactorSourceClient: DependencyKey {
 			return entities.contains(where: { !$0.isMnemonicPresentInKeychain })
 		}
 
-		let unrecoverableEntitiesCount: @Sendable () async throws -> (accounts: Int, personas: Int) = {
+		let unrecoverableEntities: @Sendable () async throws -> (accounts: [Account], personas: [Persona]) = {
 			let deviceFactorSources = try await factorSourcesClient.getFactorSources(type: DeviceFactorSource.self)
 			let entities = try await deviceFactorSources.asyncMap {
 				try await entitiesControlledByFactorSource($0, nil)
 			}
-			var accounts = 0
-			var personas = 0
+			var accounts: [Account] = []
+			var personas: [Persona] = []
```

Comment:
Might be enough to just send the addresses here?

Reply:
yeah I guess this won't be used for anything else so that's fine